device/rsvp/rsvpSessionAttribute.yaml

```diff
@@ -0,0 +1,102 @@
+components:
+  schemas:
+    Rsvp.SessionAttribute:
+      description: >-
+        Configuration for RSVP-TE SESSION_ATTRIBUTE object included in Path Messages as defined in RFC3209.
+        The bandwidth_protection_desired and node_protection_desired flags are defined in RFC4090 (Fast Reroute).
+      type: object
+      properties:
+        auto_generate_session_name:
+          description: >-
+            If this is enabled, an auto-generated Session Name is included in the SESSION_ATTRIBUTE object in the Path Message for this LSP.
+            It is suggested to include the Local IP, Remote IP, Tunnel ID and LSP ID in the auto-generated Session Name to ensure uniqueness of the name in the test.
```

Comment:
The description "It is suggested to include the Local IP, Remote IP, Tunnel ID and LSP ID in the auto-generated Session Name to ensure uniqueness of the name in the test."  is a concern here.

Shouldn't be the above description matches more to the "name" field below?